### ISSUE TITLE
Current requirements versions not compatible with python 3

### ISSUE THREAD

**Issue by wagner-certat** [xarf/python-xarf#6]
installing current master is broken because of outdated versions:

```
pip3 install -e git+git://github.com/xarf/python-xarf.git#egg=pyxarf
Obtaining pyxarf from git+git://github.com/xarf/python-xarf.git#egg=pyxarf
  Updating ./src/pyxarf clone
Collecting jsonschema==2.3.0 (from pyxarf)
  Using cached https://files.pythonhosted.org/packages/20/1c/52812523feebd744ac0e268224b5bae048b1559c88728f3628d169427947/jsonschema-2.3.0-py2.py3-none-any.whl
Collecting PyYAML==3.10 (from pyxarf)
  Using cached https://files.pythonhosted.org/packages/00/17/3b822893a1789a025d3f676a381338516a8f65e686d915b0834ecc9b4979/PyYAML-3.10.tar.gz
Collecting dnspython==1.11.1 (from pyxarf)
  Using cached https://files.pythonhosted.org/packages/f1/19/68f9cd953a45e120e75f8352d0ce1d5019e503725d89679072bbcad5f8df/dnspython-1.11.1.zip
Collecting ipaddr==2.1.11 (from pyxarf)
  Using cached https://files.pythonhosted.org/packages/08/80/7539938aca4901864b7767a23eb6861fac18ef5219b60257fc938dae3568/ipaddr-2.1.11.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-158bmxou/ipaddr/setup.py", line 19, in <module>
        import ipaddr
      File "/tmp/pip-install-158bmxou/ipaddr/ipaddr.py", line 1450
        ip_int = 0L
                  ^
    SyntaxError: invalid syntax
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-158bmxou/ipaddr/
```

please remove the strict version requirements

**Comment by FrederikP** [comment]
This is fixed in commit: bf85e0636e2819d04ad03f3ec97ab64aefd53a9e

It will be released as 0.1.0 when I had a chance to look at the other issues

**Comment by FrederikP** [comment]
Please try version 0.1.0 https://pypi.org/project/pyxarf/
